build/cqs.tsv

```diff
@@ -177,11 +170,11 @@ org.apache.parquet:parquet-encoding	1.9.0	compile
 org.apache.parquet:parquet-format	2.3.1	compile
 org.apache.parquet:parquet-hadoop	1.9.0	compile
 org.apache.parquet:parquet-jackson	1.9.0	compile
-org.apache.thrift:libthrift	0.9.3	compile
+org.apache.thrift:libthrift	0.12.0	compile
 org.apache.xml:xml-commons-resolver	1.2	compile
 org.apache.yetus:audience-annotations	0.5.0	compile
 org.apache.yetus:audience-annotations	0.7.0	compile
-org.apache.zookeeper:zookeeper	3.4.10	compile
+org.apache.zookeeper:zookeeper	3.4.14	compile
```

Comment:
Need to enter or manage to 3.4.11

Reply:
i think it's worthwhile to put in a new cq...

Reply:
New CQ for you: https://dev.eclipse.org/ipzilla/show_bug.cgi?id=21805